src/mainwindow.cpp

```diff
@@ -301,6 +330,20 @@ void MainWindow::updateMessages(int t, const QString& message, StringVec v) {
     } else {
       event_message += message;
     }
+    if (m_events.size() > 1) {
```

Comment:
it's not > 0?

Reply:
Because it's looking for repeating messages and grouping them together, represented as the latest message with a counter. (The original, unmodified messages can still be viewed in the console)